packages/app-bridge/src/react/useGuidelineActions.ts

```diff
@@ -146,7 +148,7 @@ export const useGuidelineActions = (appBridge: AppBridgeTheme) => {
     );
 
     const createDocumentGroup = useCallback(
-        async (documentGroup: CreateDocumentGroup) => {
+        async (documentGroup: Omit<DocumentGroupRequest, 'id'>) => {
```

Comment:
Please create a type instead, same for all others (`Omit<`, `RequireAtLeastOne<`)